src/player.js

```diff
@@ -517,10 +517,16 @@ export default class Player extends FakeEventTarget {
 
   /**
    * Start/resume playback.
+   * @param {boolean} programmatic - indicates the play call was not user gestured and should be handled like auto play.
    * @returns {void}
    * @public
    */
-  play(): void {
+  play(programmatic: boolean = false): void {
```

Comment:
I think adding a true/false boolean to API is not clear, this is why I suggested to add it as an object so when one calls the API it is clear what is the action: `player({programmatic: true})`